torch/lib/c10d/FileStore.cpp

```diff
@@ -310,9 +311,15 @@ std::vector<uint8_t> FileStore::compareSet(
   // it might be outdated
   pos_ = refresh(file, pos_, cache_);
   if (cache_.count(regKey) == 0) {
+    // if the key does not exist and current value is empty then set it
```

Comment:
Nit: Maybe you should re-word this comment. If the key does not exist, then there can't be any "current value". What you mean here with "current value" is the value of the `currentValue` parameter.

Reply:
Updated the comment